src/java.desktop/share/classes/javax/swing/plaf/multi/doc-files/multi_tsc.html

```diff
@@ -39,7 +39,7 @@ <h1>Using the Multiplexing Look and Feel</h1>
 <i>
 This document is based on an article
 originally published in
-<a href="http://www.oracle.com/technetwork/java/javase/tech/articles-jsp-139072.html"
+<a href="https://www.comp.nus.edu.sg/~cs3283/ftp/Java/swingConnect/archive/plaf_papers_arch/multi_update/multi_update.html"
```

Comment:
Well I am 100% sure this article wasn't originally published by the national university of singapore.
And we definitely should NOT be linking there. 
I actually think for most of these we'd be better off getting rid of the link entirely.